source/opt/ir_context.h

```diff
@@ -798,6 +806,10 @@ class IRContext {
   // Whether all specialization constants within |module_|
   // should be preserved.
   bool preserve_spec_constants_;
+
+  // DebugInfoNone instruction. We need only a single DebugInfoNone.
+  // To reuse the existing one, we keep it using this member variable.
+  Instruction* debug_info_none_;
```

Comment:
The name of this variable sounds like it should be a boolean that says if there is debug info or not.  Can you rename this and the related getter to `debug_info_none_inst_`?

Reply:
done